File: rasterizer-dotnet-examples-csharp/JpegImage.cs
using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    class JpegImage
    {
        internal static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            JpegImageFormat jpegImageFormat = new JpegImageFormat(100);
            FixedImageSize fixedImageSize = new FixedImageSize(595, 841);
            rasterizer.Draw(Program.GetOutputDocPath("JpegImage.jpg"), jpegImageFormat, fixedImageSize);
        }
    }
}

File: rasterizer-dotnet-vb/JpegImage.vb
Imports ceTe.DynamicPDF.Rasterizer

Class JpegImage
        Friend Shared Sub Run()
            Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
            Dim jpegImageFormat As New JpegImageFormat(100)
            Dim fixedImageSize As New FixedImageSize(595, 841)
            rasterizer.Draw(Program.GetOutputDocPath("JpegImage.jpg"), jpegImageFormat, fixedImageSize)
        End Sub
    End Class


